docs/editor/accessibility.md

```diff
@@ -25,6 +25,46 @@ You can adjust the Zoom level in VS Code with the **View** > **Appearance** > **
 
 When you adjust the zoom level with the **View** > **Zoom In / Out** commands, the zoom level is persisted in the `window.zoomLevel` [setting](/docs/getstarted/settings.md). The default value is 0 and each increment/decrement changes the zoom level by 20%.
 
+## Color vision accessibility
+
+You can search for extensions in **Visual Studio Marketplace** that are compatible for color vision deficiency. Use Extensions `kbstyle(Ctrl+Shift+X)` to access the Marketplace extension search bar and enter "colorblind" to populate relevant options.
```

Comment:
For keyboard shortcuts, you can use kb(commandID) instead of kbstyle. That way, the correct keybinding will be displayed on code.visualstudio.com for the readers platform. For example, to bring up the Extensions view is Ctrl+Shift+X on Windows and Linux but Cmd+Shift+X on macOS.
In this instance, you would use `kb(workbench.view.extensions)`